cloudflare.mjs

```diff
@@ -7,7 +7,9 @@ export default async function getDeploymentUrl(
   accountEmail,
   projectId,
   repo,
-  branch
+  branch,
+  environment,
```

Comment:
Is it worth introducing an `options` object for optional parameters? There may be more in the future.

Reply:
Hm, I can see how that might be nice, but it might be annoying to add another level of nesting of optional things until we also get optional chaining. 

I'm not familiar enough with babel to know how to get that working…